public/js/errorFocus.js

```diff
@@ -1,4 +1,7 @@
 
 $(document).ready(function(){
     document.getElementById('error-summary-display').focus();
```

Comment:
The error still shows up in the console (I can give you a hand checking this if you want). I think all you need to do is add the "var error =" bit in before document on line 3 & that should solve it